CITATION.cff

```diff
@@ -54,3 +54,31 @@ keywords:
 license: BSD-3-Clause
 version: 0.5.0
 date-released: '2024-05-31'
+preferred-citation:
```

Comment:
Does `preferred-citation` mean that this is preferred over the "software DOI"? Do we have a preference to cite one or the other?

Reply:
It's the citation that will appear on GitHub under the "Cite this repository". Unfortunately I don't know of a way of adding the paper to `CITATION.cff` without using the `preferred-citation` field. (Ideally, both the software and the paper should be cited.)

Reply:
I would prefer to keep the software DOI as primary citation, as papers and posters can get outdated very soon.
We might add other references to the README